powerauth-restful-server-javaee/src/main/java/io/getlime/security/powerauth/app/rest/api/javaee/controller/v2/CustomActivationController.java

```diff
@@ -138,7 +134,7 @@
             return powerAuthApiResponse;
 
         } catch (Exception ex) {
-            logger.warn(ex.getMessage(), ex);
+            logger.warn("Create activation failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.